src/display/components/total_bandwidth.rs

```diff
@@ -20,10 +20,12 @@ impl<'a> TotalBandwidth<'a> {
             } else {
                 Color::Green
             };
+            let interface = datalink::interfaces().into_iter().map(|interface|interface.name).collect::<Vec<String>>();
```

Comment:
So, we actually already do something similar on the OS layer. Since we have this information already, I think it would be best if we pass it to the UI in order not to have to do the query a second time (and possibly get different results if something has changed on the system while the app was running, for example).

I think the place to pass this information to the UI would be here: https://github.com/imsnif/bandwhich/blob/master/src/main.rs#L117